content/blog/2018/02/2018-02-22-cheetah.adoc

```diff
@@ -0,0 +1,134 @@
+---
+:layout: post
+:title: "Project Cheetah - Faster, Leaner Pipeline That Can Keep Up With Demand"
+:tags:
+- pipeline
+- performance
+- scalability
+- features
+:author: svanoort
+---
+
+:toc: 
+
+== The Problem To Solve
+Since it launched, Pipeline has had a bit of a Dr. Jekyll and Mr. Hyde performance problem.  In certain circumstances, Pipeline turns from a mild-mannered CI/CD assistant into a monster.  It will happily eat storage read/write capacity like popcorn with nary a care for the other concerns of our friendly butler.  When combined with additional factors, this results in real-world stability problems.  For example, combining slow storage with a spike in running Pipelines has brought down production Jenkins at more than one company.  Similarly users see issues if a busy master gets hit with an extra source of stress; past culprits have been heavy (ab)use of automated APIs, now-solved user lookup bugs, backup jobs, and plugins run amuck that load excessive numbers of builds.
+
+Now I'm _not_ saying this to scare people or to criticize the technology I've helped build. Implementing link:/blog/2017/02/01/pipeline-scalability-best-practice/[Pipeline scalability best practices] coupled with SSD storage keeps Jenkins in a happy place.  We just need context on the weaknesses to see why it's important to address them.
+
+== Introducing Project Cheetah
+
+Today we're announcing the first major results of Project Cheetah, our long-running effort to address these challenges and improve Pipeline scalability.  More broadly, Cheetah aims to help in 3 places:
```

Comment:
This paragraph could be improved by pointing out what version of what plugin these changes were released in. That will help connect the dots between the code name and the actual results a reader can realize in their Jenkins environment.